default/main.py

```diff
@@ -51,8 +51,14 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                         ssl=ssl_context
                     ) as resp:
                     json = await resp.json()
+                    json['api_key'] = user.apikey
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    httpSession.post(user.webhook, data=json)
+                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
+                        post_response = await post.text()
+                        if post.status != 200:
```

Comment:
Should this be `post_response.status` instead?

Reply:
No, that's correct... Maybe it's not a good naming